include/roboteam_ai/stp/evaluations/position/GoalShotEvaluation.h

```diff
@@ -0,0 +1,43 @@
+//
+// Created by maxl on 23-03-21.
+//
+
+#ifndef RTT_GOALSHOTEVALUATION_H
+#define RTT_GOALSHOTEVALUATION_H
+
+#include <NFParam/Param.h>
```

Comment:
? do we really still use this?

Reply:
This one in particular not. When re-adding the plays into the AI (next step) a more of the old invariants will pop out as obsolete, I'll remove them then.